internal/relationtuple/swagger_definitions.go

```diff
@@ -17,13 +17,13 @@ var (
 
 // The patch request payload
 //
-// swagger:parameters patchRelationTuples
+// swagger:parameters patchRelationships
 type patchPayload struct {
 	// in:body
 	Payload []*ketoapi.PatchDelta
```

Comment:
This should also use a "relationship" related naming.

Reply:
I'm not sure what you mean. It is used in `patchRelationships`.

Reply:
`PatchDelta` is a generic name, but refers to a `RelationshipPatchDocument`. The problem is that in the generated OpenAPI  spec, `PatchDelta` appears. I think this should get the appropriate swagger annotation:

```
// swagger:model relationshipPatch
type PatchDelta {
```